src/Gui/ToolBarManager.cpp

```diff
@@ -294,10 +304,137 @@ class ToolBarArea : public QWidget
     boost::signals2::scoped_connection &_conn;
 };
 
+class ToolBar: public QToolBar
+{
+public:
+    void initStyleOption(QStyleOptionToolBar *option) const
+    {
+        QToolBar::initStyleOption(option);
+    }
+};
+

```

Comment:
While I do understand benefit of having our own ToolBar widget (it would come handy for a lot of reasons) - I don't understand why you create this class now while its only method is a simple call to parent implementation. I'd suggest removing this method and adding other useful helpers like `float()`. Override the moveable behavior etc.